libshaderc_util/src/compiler.cc

```diff
@@ -105,18 +105,13 @@ namespace shaderc_util {
 
 unsigned int GlslangInitializer::initialize_count_ = 0;
 std::mutex* GlslangInitializer::glslang_mutex_ = nullptr;
+std::once_flag GlslangInitializer::initialize_lock_flag_;
```

Comment:
So this is a class object with static duration.
The style guide warns us about these.  https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

So is this class trivially destructible?